content/posts/client-mpv.md

```diff
@@ -0,0 +1,64 @@
++++
+title = "Client Spotlight: MPV"
+subtitle = "Watch all your favorite content without transcoding!"
+author = “Ian Walton”
+githubusername = "iwalton3"
+date = "2020-02-07"
+justify = "center"
++++
+
+Jellyfin MPV Shim is a lightweight cast-only client that allows you to cast videos from Jellyfin to the <a href="https://mpv.io/">MPV Media Player</a>.
+<!--more-->
+It runs in the system tray and stays out of your way until you want to play something.
+
+It also has excellent codec support and many configurable options for advanced users.
+
+
+If you watch a lot of anime, this client is definitely for you.
+
+The player has full support for 10-bit HEVC video with subtitles.
+
+Additionally, there is a menu option that allows setting subtitles and audio preferences over an entire season of TV at once.
+
+
+<img src="/images/mpv-shim/blender.png" name="Interface" />
+
+
+To get started on Windows, simply <a href="https://github.com/iwalton3/jellyfin-mpv-shim/releases">download the binary</a> and run it.
+
+If you are on Linux, <a href="https://github.com/iwalton3/jellyfin-mpv-shim/blob/master/README.md#linux-installation">follow the instructions here</a>.
+
+After you have started the client, enter the URL of your server and the username and password to connect.
+
+The client will run in a tray icon, which allows you to access the configuration and view the error logs.
+
+
+Casting media is easy.
+
+Open the menu in the Jellyfin web or mobile clients and select Jellyfin MPV Shim.
```

Comment:
Maybe this should be "casting menu" or "Play On" menu?  There's several menus spread around, could be confusing